src/views/remove-authorised-person.njk

```diff
@@ -36,12 +36,21 @@
             <p>{{ lang.if_you_remove + user_details + lang.digital_authorisation_this_means_they + companyName + lang.without_a_current_auth_code + user_details + lang.to_let_them_know_you_have_removed }}</p>
 
             {% set warning_text_html %}
-            <span class="govuk-!-display-block govuk-!-margin-bottom-5">
-                {{ user_details + lang.will_still_be_able_to_file }}
-            </span>
-            <span class="govuk-!-display-block">
-                {{ lang.you_may_wish_to_change_the_auth_code + user_details + lang.digital_authorisation }}
-            </span>
+                {% if lang.remove == "Dileu awdurdodiad " %}
```

Comment:
I think the current language is available in the nunjucks templates - its set to 'locale' if you are just checking the current language here. So I think something like this might work - 

   {% if locale == “cy” %}
     …
    {% else %}